api/odk/helpers/aggregate-submissions.js

```diff
@@ -5,9 +5,34 @@ var async = require('async');
 
 var ASYNC_LIMIT = 10;
 
-module.exports = function (formName, errorCallback, aggregateCallback) {
+module.exports = function (req, errorCallback, aggregateCallback) {
```

Comment:
Since this is a "helper", I would keep `req` out of the signature and move `limit` and `offset` args just before the callback(s), allowing it to be more cleanly used elsewhere.